homeassistant/components/iaqualink/__init__.py

```diff
@@ -191,6 +210,14 @@ async def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+async def try_await(awaitable: Awaitable) -> None:
```

Comment:
Please name this to show intent that it converts AqualinkServiceException to HomeAssistantError

Reply:
Any suggestion? wait_or_reraise()?

Reply:
`await_or_reraise` should good to me